rsocket/RSocketServer.cpp

```diff
@@ -82,6 +84,7 @@ void RSocketServer::acceptConnection(
     std::unique_ptr<DuplexConnection> connection,
     folly::EventBase&,
     std::shared_ptr<RSocketServiceHandler> serviceHandler) {
+  stats_->serverConnectionAccepted();
```

Comment:
Should we place this line at the inside of onRSocketSetup function, or onRSocketResume?

Reply:
I think this particular event is really just about getting an instance of DuplexConnection, the next step is counters in onSetup, onResume, error, ... which we hopefully have. 
I would leave this where it is.